### PR TITLE
delete a single record

### PR DESCRIPTION
**Pull request by AnnaLangner**
I wrote a script which removes the "picture " record from the json file

### CODE REVIEW

**Review by klangner — COMMENTED** [review]
There are no changes.

**Comment by AnnaLangner** [comment]
Sorry, I think I missed something earlier

**Review by klangner — COMMENTED** [review]
Do not store binary file in the repo. Add content of /db to .gitignore

File: script/remove_record.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import json
+
+
+JSON_FILE = 'init/persons-test.json'
+persons = json.load(open(JSON_FILE, encoding='utf-8'))
+
+
+def remove_record_from_json():
```
Comment (klangner) [inline script/remove_record.py]: * Add main function
* Function should do 1 thing only
* do not use global variables

**Review by klangner — CHANGES_REQUESTED** [review]

**Review by klangner — APPROVED** [review]

**Pull request merged**